getting-started/analyst-exercise/index.md

```diff
@@ -5,7 +5,7 @@ active: getting-started
 ---
 
 ## Setup
-Consult this [FireEye report](report) for a prose description of malicious actors from Iran targeting governments and commercial computers.
+Consult this [FireEye report](report.html) for a prose description of malicious actors from Iran targeting governments and commercial computers.
```

Comment:
Hmmm, they are both working for me now (at least on stixproject.github.io). Without the .html doesn't work for me locally. I wonder if GitHub doesn't remove old pages.